### ISSUE TITLE
Strict axis checking in PrePARE

### ISSUE THREAD

**Issue by aradhakrishnanGFDL** [PCMDI/cmor#399]
PrePARE to check the following. 

-If the variable in the input datasets confines to the CMOR standards (and using CMIP6 Model Output Requirements from Karl et al)  in terms of axis and dimensionality. This includes an optional high-level basic checks, with the following being the wish-list/use-cases:

1. checking 2D, 3D axis names ; 
2. singleton dimension names ; 
3. model level
4. pressure level fields
5. unstructured grids, corresponding coordinate attributes/axis. 

Recording this as an issue after an offline conversation with Karl, Charles.

**Comment by doutriaux1** [comment]
Thanks Aparna

**Comment by doutriaux1** [comment]
@mauzey1 this is just for you to monitor. You don't have to do anything about this yet.

**Comment by mauzey1** [comment]
@taylor13 Should this just be another list of features for future PrePARE versions?

**Comment by taylor13** [comment]
Yes.  If you note them somewhere else, this issue can be closed.

**Comment by taylor13** [comment]
Before rewriting PrePARE, we'll need to write down a prioritized lists of things we want to check.  We can then see if this one is high enough priority to include in version 4.0

**Comment by mauzey1** [comment]
Since we are deprecating PrePARE in a future release of CMOR, this issue is now obsolete.
